### PR TITLE
Input sanitizing

### PR DESCRIPTION
**Pull request by robthethird**
Input should be safe across the ui now.

### CODE REVIEW

File: src/tables/LocationTable.java [hunk 1]
```diff
@@ -62,6 +62,20 @@ public static void populateFromCSV(Connection conn) {
         SQLHelper.execute(conn, sb.toString());
     }
 
+    public static boolean isValidZip(Connection conn, Integer zipCode) {
+        String query = "SELECT 1 FROM location WHERE zip_code = " + zipCode + ";";
+
+        ResultSet resultSet = SQLHelper.executeQuery(conn, query);
+        try {
+            if (resultSet.next()) {
+                return true;
```
Comment (jarrydlee) [inline src/tables/LocationTable.java]: Return resultset.next()?

**Comment by jarrydlee** [comment]
Apart from that one comment, looks good

**Comment by robthethird** [comment]
Good call I'll change that.

File: src/tables/LocationTable.java (hunk 1 above)
Comment (ClayBenson94) [inline src/tables/LocationTable.java]: Fixed it in the more recent commit.

**Pull request merged**